heron/executor/tests/python/heron_executor_unittest.py

```diff
@@ -105,50 +105,41 @@ def build_packing_plan(self, instance_distribution):
            "-cp metricsmgr_classpath org.apache.heron.metricsmgr.MetricsManager " \
            "--id=metricsmgr-%d --port=metricsmgr_port " \
            "--topology=topname --cluster=cluster --role=role --environment=environ " \
            "--topology-id=topid " \
            "--system-config-file=%s --override-config-file=%s " \
-           "--sink-config-file=metrics_sinks_config_file" %\
-           (container_id, container_id, INTERNAL_CONF_PATH, OVERRIDE_PATH)
+           "--sink-config-file=metrics_sinks_config_file" % \
+           (container_id, INTERNAL_CONF_PATH, OVERRIDE_PATH)
 
   def get_expected_metricscachemgr_command():
     return "heron_java_home/bin/java -Xmx1024M -XX:+PrintCommandLineFlags " \
-           "-Djava.net.preferIPv4Stack=true -verbosegc " \
-           "-XX:+UseConcMarkSweepGC -XX:+CMSScavengeBeforeRemark -XX:TargetSurvivorRatio=90 " \
-           "-XX:+PrintGCDetails -XX:+PrintGCTimeStamps -XX:+PrintGCDateStamps " \
-           "-XX:+PrintGCCause -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=5 " \
-           "-XX:GCLogFileSize=100M -XX:+PrintPromotionFailure -XX:+PrintTenuringDistribution " \
-           "-XX:+PrintHeapAtGC -XX:+HeapDumpOnOutOfMemoryError -XX:ParallelGCThreads=4 " \
-           "-Xloggc:log-files/gc.metricscache.log " \
+           "-Djava.net.preferIPv4Stack=true " \
+           "-XX:+UseG1GC -XX:+ParallelRefProcEnabled -XX:+UseStringDeduplication " \
+           "-XX:MaxGCPauseMillis=100 -XX:InitiatingHeapOccupancyPercent=30 " \
+           "-XX:+HeapDumpOnOutOfMemoryError -XX:ParallelGCThreads=4 " \
            "-cp metricscachemgr_classpath org.apache.heron.metricscachemgr.MetricsCacheManager " \
            "--metricscache_id metricscache-0 --server_port metricscachemgr_serverport " \
            "--stats_port metricscachemgr_statsport --topology_name topname --topology_id topid " \
            "--system_config_file %s --override_config_file %s " \
            "--sink_config_file metrics_sinks_config_file " \
-           "--cluster cluster --role role --environment environ" %\
-           (INTERNAL_CONF_PATH, OVERRIDE_PATH)
+           "--cluster cluster --role role --environment environ" \
+           % (INTERNAL_CONF_PATH, OVERRIDE_PATH)
 
   def get_expected_healthmgr_command():
     return "heron_java_home/bin/java -Xmx1024M -XX:+PrintCommandLineFlags " \
-           "-Djava.net.preferIPv4Stack=true -verbosegc " \
-           "-XX:+UseConcMarkSweepGC -XX:+CMSScavengeBeforeRemark -XX:TargetSurvivorRatio=90 " \
-           "-XX:+PrintGCDetails -XX:+PrintGCTimeStamps -XX:+PrintGCDateStamps " \
-           "-XX:+PrintGCCause -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=5 " \
-           "-XX:GCLogFileSize=100M -XX:+PrintPromotionFailure -XX:+PrintTenuringDistribution " \
-           "-XX:+PrintHeapAtGC -XX:+HeapDumpOnOutOfMemoryError -XX:ParallelGCThreads=4 " \
-           "-Xloggc:log-files/gc.healthmgr.log " \
+           "-Djava.net.preferIPv4Stack=true " \
```

Comment:
"-Xloggc:log-files" might be necessary.

Reply:
Same as comment above. I believe `-Xloggc:log-files` is a GC specific setting. This does not impact the default logging.